ComposeAdvanced/README.md

```diff
@@ -1,3 +1,5 @@
+This project is now deprecated and unmaintained.
```

Comment:
Is it worth giving either 
a) a brief reasons
b) a link

If we remove from main, probably not.  But figured worth asking.

Reply:
you are totally right, worth linking to Horologist and to ComposeStarter, will add as part of this PR.